framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -100,6 +103,11 @@
         implements TransferFunction<V, S> {
 
     /**
+     * Indicates that the whole-program inference is on.
```

Comment:
Please change occurrences of "whole-program" to "signature", so that the feature is consistently named.